fuzzers/ankou/fuzzer.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration code for AFL fuzzer."""
+
+import shutil
+import subprocess
+import os
+
+from fuzzers import utils
+
+# OUT environment variable is the location of build directory (default is /out).
+
+
+def prepare_build_environment():
```

Comment:
You don't really need to define this function, we do it in the AFL fuzzer.py because it is useful for people who import it.